app/assets/stylesheets/nfg_ui/network_for_good/admin/nfg_theme/custom/_bee.scss

```diff
@@ -5,11 +5,16 @@
 }
 
 // Height if just the builder-header is present
-.builder-header + .builder-container {
+.builder-header ~ .builder-container {
   #bee_plugin_container { height: calc(100vh - 72px); }
 }
 
 // Height if both builder-header and builder-nav are present (typical in onboarder)
-.builder-header + .builder-nav + .builder-container {
+.builder-header ~ .builder-nav ~ .builder-container {
   #bee_plugin_container { height: calc(100vh - 180px); }
+
+  // Height if there are buttons above the bee plugin
+  .builder-container-buttons {
+    ~ #bee_plugin_container { height: calc(100vh - 284px); }
```

Comment:
@kylejmorin have we confirmed this is backwards compatible? Also is this function running at compile or precompile time? See this stackoverflow answer: https://stackoverflow.com/a/25547896

Reply:
@kylejmorin i'm assuming since we've been using the calc function for forever, that this has been thoroughly tested... yeah just checking in on it.

Reply:
@jonathanroehm yep it is good to go. I haven't seen any issues with the calc() on any of the browsers I use. We have a fallback further up in the file that if calc() doesn't work, then it defaults to a height of 600px.